openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -613,6 +614,7 @@ def test_nonexistent_course(self):
 
     @override_waffle_flag(COURSE_HOME_USE_LEGACY_FRONTEND, active=True)
     @override_waffle_flag(COURSE_PRE_START_ACCESS_FLAG, active=True)
+    @override_settings(PLATFORM_NAME="edX")
```

Comment:
Are you able to do this here:

```
@override_settings(PLATFORM_NAME=settings.PLATFORM_NAME)
```

And would that be better?

Reply:
Or does that defeat the whole purpose if you're trying _not_ to use the original setting... 🤔

Reply:
The issue is that the test is comparing data that has `edX` hardcoded as the platform name. I could change the data to use `Your Platform Name Here`, but that feels more brittle if someone has something else set for PLATFORM_NAME locally then they get random test failures. Also, this is the pattern I see throughout edx-platform.